lab-miketreat/test/module-test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
If this is your index.js file, you need to require in the file that holds your logic for reading in a file and invoke that function.